kinesis_video_streamer/package.xml

```diff
@@ -13,11 +13,11 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <depend>aws_common</depend>
-  <depend>aws_ros1_common</depend>
-  <depend>kinesis_manager</depend>
+  <depend version_lt='2.0.0'>aws_common</depend>
```

Comment:
Why are we changing package.xml in release?

Reply:
this is adding version tags in the dependencies for this package so that when it builds, this branch uses version2 of its dependencies.

Reply:
Release branch should have version qualifiers (assuming rosdep works as intended when it comes to `apt install`ing a desired version).

Master branch should not have version qualifiers, because it should always take latest (technically it should have sources of its dependencies instead of binaries from `apt`).

In this case @xabxx, I think it's supposed to be `version_lt='3.0.0'` for this release-v2 branch.

Reply:
^^what miaofei said and one, changed to 3.0.0.

Reply:
@xabxx @mm318 

> Release branch should have version qualifiers (assuming rosdep works as intended when it comes to `apt install`ing a desired version).

Perhaps (see note below), but we should never modify code in released branches. This is a divergence from the version that is already in apt. The only permitted changes are to travis and README files.

*The version qualifer doesn't do anything, and it is very strange to specify a `version_lt` tag before a newer version has been released. It is counter productive: the future version 3.0.0 of aws_common might actually be compatible with this version of the kinesis streamer, we don't know yet. `version_gt` would make much more sense.

Reply:
I think it would make sense to have both. Most other package managers lock to a major version.

Reply:
It's not scalable for us in terms of development and testing resources to turn the possibility of a future version being compatible into a customer guarantee that the future version is compatible and supported.